ShowMyData/Pages/Index.cshtml.cs

```diff
@@ -12,19 +12,25 @@ namespace ShowMyData.Pages
 {
     public class IndexModel : PageModel
     {
-        public void OnGet()
+        public JsonResult OnGet()
+        {
+
+            string policeCrimeString = getJsonData("https://data.cincinnati-oh.gov/resource/ksej-uzjq.json");
```

Comment:
Changed variable names for better understanding

Reply:
Good thought... you don't need to include the variable type in the variable name.